util/install

```diff
@@ -171,10 +177,13 @@ copy_music()
   for track_number in 1 2 3; do
     local cd_number=$(($track_number+1))
     local track_wav="$TARGET/music/track_$track_number.wav"
+    local track_flac="$TARGET/music/track_$track_number.flac"
     local track_ogg="$TARGET/music/track_$track_number.ogg"
-    "$CDPARANOIA" "$cd_number" "$track_wav" || return 1
+    "$CDPARANOIA" -d "$DEVICE" "$cd_number" "$track_wav" || return 1
```

Comment:
From `cdparanoia --help`:

```
  -d --force-cdrom-device   <dev> : use specified device; disallow
                                    autosense
```

The device parameter should not be mandatory, and therefore adding '-d' to cdparanoia should be conditional.